<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="9" hostname="DESKTOP-5JJETEC" failures="1" tests="1" name="Default test" time="8.209" errors="0" timestamp="2023-01-10T12:12:39 IST">
  <testcase classname="TestCases.LoginPageTest" name="verifyTitleTest" time="2.03">
    <failure type="java.lang.AssertionError" message="expected [Swag Labs] but found [1Swag Labs]">
      <![CDATA[java.lang.AssertionError: expected [Swag Labs] but found [1Swag Labs]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at TestCases.LoginPageTest.verifyTitleTest(LoginPageTest.java:32)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:568)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)
at org.testng.SuiteRunner.run(SuiteRunner.java:276)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1134)
at org.testng.TestNG.runSuites(TestNG.java:1063)
at org.testng.TestNG.run(TestNG.java:1031)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
    </failure>
  </testcase> <!-- verifyTitleTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyProductTitleTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyProductTitleTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyErrorMessageForPasswordTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyErrorMessageForPasswordTest -->
  <testcase classname="TestCases.LoginPageTest" name="loginCredentialTest" time="0.0">
    <ignored/>
  </testcase> <!-- loginCredentialTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyLoginLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyLoginLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyPeekLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyPeekLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyBotLogoTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyBotLogoTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyCurrentUrlTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyCurrentUrlTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyErrorMessageForUsernameTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyErrorMessageForUsernameTest -->
  <testcase classname="TestCases.LoginPageTest" name="verifyErrorMessageTest" time="0.0">
    <ignored/>
  </testcase> <!-- verifyErrorMessageTest -->
</testsuite> <!-- Default test -->
